packages/vue/src/components/atoms/SfIcon/SfIcon.vue

```diff
@@ -43,6 +36,16 @@ import icons from "@storefront-ui/shared/icons/icons";
 import { iconColorsValues as SF_COLORS } from "@storefront-ui/shared/variables/colors";
 import { sizesValues as SF_SIZES } from "@storefront-ui/shared/variables/sizes";
 const SF_ICONS = Object.keys(icons);
+
+const fillPathUrl = (index) => `url(#${index})`;
+
+/**
+ * 1. Aria-hidden on by default
```

Comment:
comments for to do? :)

Reply:
oops. Yes :) @justyna-13  Removing it now :)